microsoft-edge/toc.yml

```diff
@@ -1338,6 +1338,33 @@
                   href: /microsoft-edge/webview2/reference/win32/index
                   displayName: functions, methods
 
+            - name: JavaScript
```

Comment:
As mentioned by Andy, we should consider how to make this clear that this is not a way to create WebView2 with JS APIs, but rather specific to some WebView2 APIs that exist in JS.

Reply:
Request on change

Reply:
Its hard to get this across. Some ideas:
* JavaScript APIs in WebView2
* JavaScript Support APIs
* JavaScript Supplement APIs
* JavaScript Addition APIs

Are there any examples out there to take inspiration from?

Reply:
This probably has my vote from your list above? "JavaScript APIs in WebView2"